quasar/main.tf

```diff
@@ -615,3 +615,60 @@ module "storage" {
   acl        = "private"
   versioning = true
 }
+
+# Fivetran logging settings based on https://fivetran.com/docs/logs/cloudwatch/setup-guide
+# and https://github.com/tropos-io/tf-fivetran-cloudwatch
+
+data "aws_iam_policy_document" "fivetran_cloudwatch_integration_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::834469178297:root"]
```

Comment:
This is Fivetran's AWS account ID, right? Can we add a comment on this line noting that?